docker-images/sw360empty/Dockerfile

```diff
@@ -21,4 +21,4 @@ COPY tomcatdeploy.sh /usr/local/bin/tomcatdeploy.sh
 COPY docker-entrypoint.sh /usr/local/bin/entrypoint.sh
 
 ENTRYPOINT ["/usr/local/bin/entrypoint.sh"]
-CMD ["tail", "-f", "/opt/sw360/tomcat-9.0.17/logs/catalina.out"]
```

Comment:
does this get the following env variable?

https://github.com/sw360/sw360chores/blob/35f7dbeb49a1fb012db0df21219f6f9ec5c2ef06/docker-images/sw360empty/docker-entrypoint.sh#L54

Reply:
Yes.